src/components/EditDrawer.js

```diff
@@ -145,38 +190,21 @@ export default function EditDrawer({ open, setOpen, editItem, setEditItem }) {
         editorProps={{
           displayIndentGuides: false,
         }}
-        value={
-          typeof editItem.value === 'object'
-            ? JSON.stringify(editItem.value, null, 4)
-            : deserialize(editItem.value, false)
-        }
+        value={editorValue}
+        // value={
+        //   typeof editItem.value === 'object'
+        //     ? JSON.stringify(editItem.value, null, 4)
+        //     : deserialize(editItem.value, false)
+        // }
         theme="tomorrow_night_bright"
         name="test editor"
-        onChange={newText => {
-          try {
-            let newValue
-
-            console.log('LOG: editItem.valueType', editItem.valueType)
-            if (editItem.valueType === 'jsx') {
-              // transform the JSX here
-              // eslint-disable-next-line no-undef
-              newText = Babel.transform(newText, {
-                presets: ['es2015', 'react'],
-                parserOpts: { sourceType: 'script', minified: true },
-              }).code
-              newValue = newText ? eval(newText) : undefined // eslint-disable-line
-              newValue = newText
-            } else {
-              newValue = newText ? eval(`() => (${newText})`)() : undefined // eslint-disable-line
-            }
-
-            const serialized = serialize(newValue)
-            updatePropState(editItem.propName, serialized) // eslint-disable-line
-            setEditItem({ ...editItem, value: newText })
-          } catch (e) {
-            console.error(e)
-          }
+        onChange={newValue => {
+          console.log('LOG: newValue', newValue)
+          setEditorValue(newValue)
         }}
+        // onBlur={(event, editor) => {
+          
+        // }}
```

Comment:
Do we need the onBlur for the babel stuff later? I feel like this is left over from our refactor of the edit drawer